app/presenters/publishing_api_presenters/html_attachment.rb

```diff
@@ -1,6 +1,11 @@
 require_relative "../publishing_api_presenters"
 
 class PublishingApiPresenters::HtmlAttachment < PublishingApiPresenters::Item
+  def initialize(item, update_type: nil)
+    super
+    item.govspeak_content.render_govspeak!
```

Comment:
Only problem with that is that the `headings` method requires that the the Govspeak has been rendered too so that would require ensuring they are called in the correct order, or adding complexity with some sort of `render_unless_already_rendered` shenanigans.